src/main/java/dk/cngroup/lentils/dto/CodewordDTO.java

```diff
@@ -1,6 +1,6 @@
-package dk.cngroup.lentils.entity.formEntity;
+package dk.cngroup.lentils.dto;
 
-public class Codeword {
+public class CodewordDTO {
```

Comment:
Paráda. Jen to prosím změň ještě na CodewordFormDTO, jak to máš i u těch ostatních.

Reply:
Docela mě to psychicky bolí vyhledávat kde je codeword z Cypher a kde tento Codeword :D

Reply:
Ale teď už to bude jednoduché vlastně :)